app/controllers/concerns/foreman/controller/parameters/template_params.rb

```diff
@@ -34,7 +34,27 @@ def ui_template_export_params
         end
 
         def template_import_params
-          add_taxonomy_params(base_import_params(:none))
+          transform_lock_param add_taxonomy_params(base_import_params(:none))
+        end
+
+        def transform_lock_param(params)
+          lock = params[:lock]
+          return params if lock.nil?
+
+          if lock == "true" || lock.is_a?(TrueClass) || lock == "1"
```

Comment:
This is okay, but wouldn't be safer to compare strings? E.g. `lock.to_s == '1'` in case if lock is integer?